pkg/plugin/plugins.go

```diff
@@ -292,6 +292,15 @@ func (c Cache) RegisterPostHooks(ctx context.Context, id int, hookType HookTrigg
 	})
 }
 
+func (c Cache) ExecuteGalleryUpdatePostHooks(ctx context.Context, input models.GalleryUpdateInput, inputFields []string) {
```

Comment:
Not sure about that one.
I duplicated the Scene logic.
Everything seems to be there to make it work.